### PR TITLE
fix(utils): add writeInteraction types and update base64url logic, disable bunding on all writeInteractions

### PR DESCRIPTION
**Pull request by dtfiedler**

### CODE REVIEW

File: src/common/contracts/warp-contract.ts [hunk 1]
```diff
@@ -89,12 +87,14 @@ export class WarpContract<T>
     }
     const warpSigner = new Signature(this.warp, {
       signer: async (tx: Transaction) => {
+        const publicKey = await signer.publicKey;
```
Comment (atticusofsparta) [inline src/common/contracts/warp-contract.ts:90]: do we need to call set public key for arconnect signer?

**Review by atticusofsparta — APPROVED** [review]

File: src/common/contracts/warp-contract.ts (hunk 1 above)
Comment (dtfiedler) [reply src/common/contracts/warp-contract.ts:90]: https://github.com/ar-io/ar-io-sdk/pull/57/files#diff-6322e9c5f09198a48852508064d79447ceeb2f052204577f63789722aff64a3bL88

**Review by atticusofsparta — DISMISSED** [review]

**Pull request merged**